### ISSUE TITLE
Feature request: Error events

### ISSUE THREAD

**Issue by nacalai** [DmitriySalnikov/OBSNotifier#6]
Hey there. Really loving this plugin you made. It's great! Totally love with the nvidia theme you got going.

I'm not too code-savvy, otherwise I would've tried implementing this myself. 

It would be nice if you could add notifications based on dropped frames, encoder lag, and the similiar errors one might get.

Also; a bit on the side, but, I would love to create custom notifications that could be triggered via GET or similiar. I'm sure if I spend two weeks, I might come up with something. Perhaps you're able to provide a small guideline of how something similiar can be accomplished? The developer intro in readme doesn't get me far :/

**Comment by DmitriySalnikov** [comment]
Yes, I also thought about similar events. But I'm not streaming yet, and I usually record videos without any problems, so I didn't have a good reason to implement these events.

> Also; a bit on the side, but, I would love to create custom notifications that could be triggered via GET or similiar.

I don't quite understand. Do you want to implement a [custom Event](https://github.com/obsproject/obs-websocket/blob/master/docs/generated/protocol.md#customevent) in your plugin? I don't even know how they are created yet.. 🤷‍♂️

You can also look at [NvidiaNotification.cs](https://github.com/DmitriySalnikov/OBSNotifier/blob/master/Plugins/NvidiaLikeNotification/NvidiaNotification.cs), this can be regarded as an example of the implementation of the plugin interface.

(In fact, I should reconsider how the previews is displayed, because for now it works a little tricky)

**Comment by nacalai** [comment]
I think these streaming notifications would be a huge "sales point" for this plugin. I have no use of any of the notifications. I just love the design. It would be a must have plugin if it pop ups "Dropped frames" in a fancy way on a monitor. I have other ways of doing it, but, this plugin is pretty to look at, and easy setup, so it would be more convenient for people :)

As for custom notifications, an example use case is in conjuction with Advanced Scene Switcher, which is an automation plugin for OBS. I could have a macro where a trigger could have an action that sends a custom notification via your plugin.

But when I think about it, perhaps it is better the opposite way. The developer of ASS (convenient name) could make the notifications prettier, like yours. But..obviously that makes your plugin less attractive, so to speak. Just sharing some thoughts!

**Comment by DmitriySalnikov** [comment]
> would be a huge "sales point"

yeah, it would be nice to get more than $4 in a year for this app...

At the moment I'm busy with other things, I don't know when I'll add the `Dropped frames` event, maybe soon, maybe not.
